graphene_sqlalchemy/types.py

```diff
@@ -21,70 +24,91 @@
 from .utils import get_query, is_mapped_class, is_mapped_instance
 
 
+class ORMField(OrderedType):
+    def __init__(
+        self,
+        type=None,
+        prop_name=None,
+        description=None,
+        deprecation_reason=None,
+        required=None,
+        _creation_counter=None,
+        **field_kwargs
+    ):
+        super(ORMField, self).__init__(_creation_counter=_creation_counter)
+        # The is only useful for documentation and auto-completion
+        common_kwargs = {
+           'type': type,
+           'prop_name': prop_name,
+           'description': description,
+           'deprecation_reason': deprecation_reason,
+           'required': required,
+        }
+        common_kwargs = {kwarg: value for kwarg, value in common_kwargs.items() if value is not None}
+        self.kwargs = field_kwargs
+        self.kwargs.update(common_kwargs)
+
+
 def construct_fields(
     obj_type, model, registry, only_fields, exclude_fields, connection_field_factory
 ):
     inspected_model = sqlalchemyinspect(model)
+    all_model_props = OrderedDict(
```

Comment:
This seems inefficient and potentially inflexible. Could we use [Mapper.attrs](https://docs.sqlalchemy.org/en/13/orm/mapping_api.html#sqlalchemy.orm.mapper.Mapper.attrs) or [Mapper.all_orm_descriptors](https://docs.sqlalchemy.org/en/13/orm/mapping_api.html#sqlalchemy.orm.mapper.Mapper.all_orm_descriptors) instead of combining all of these, which I think are subsets of the aforementioned?

Reply:
Fair enough.